gradle.properties

```diff
@@ -32,4 +32,3 @@ systemProp.http.proxyPort=0
 systemProp.http.proxyHost=
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableR8=false
```

Comment:
Is this change really needed?

Reply:
enableR8 is deprecated so if don't remove it, we can't build the project, it throws the build error

Reply:
That's why i remove that line

Reply:
Resolved